contracts/interfaces/IOnChainVault.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
Why the import?

Reply:
cause in init function there is IERC20 as func input